src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        figures[3] = new Square("black", "", 5);
+        figures[4] = new RightTriangle("yellow", "", 5, 4);
```

Comment:
The constructors for `Square` and `RightTriangle` are being called with an empty string as the second parameter. Ensure that this matches the expected parameters for each figure's constructor. If the second parameter is not supposed to be an empty string, you should provide the appropriate value.